lib/main.dart

```diff
@@ -1,6 +1,7 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:groovin_widgets/groovin_widgets.dart';
```

Comment:
Question: what is this used for?

Reply:
This is for the `ModalDrawerHandle()` widget used in the bottom sheet. It adds the little handle looking thing :)